ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -854,17 +846,54 @@ <h3 class="title">
 
   <div class="save-edits">
     <mat-divider *ngIf="isEditMode || isPkEditMode || isFkEditMode"></mat-divider>
+
     <div class="save-button" *ngIf="isEditMode && currentTabIndex === 0">
-      <button mat-raised-button color="primary" (click)="toggleEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveColumnTable()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isPkEditMode && currentTabIndex === 1">
-      <button mat-raised-button color="primary" (click)="togglePkEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="savePk()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="togglePkEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isFkEditMode && currentTabIndex === 2">
-      <button mat-raised-button color="primary" (click)="toggleFkEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveFk()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleFkEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isIndexEditMode && currentTabIndex === -1">
-      <button mat-raised-button color="primary" (click)="toggleIndexEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveIndex()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleIndexEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
```

Comment:
After interleaving a table then editing primary key to drop the column that was used in interleaving then adding it back is not showing interleave suggestion